backend/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.7-slim
 ADD tools /src/tools
 ADD backend /src/backend
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/backend && python setup.py install
+RUN cd /src/tools && python --disable-pip-version-check --no-cache-dir --quiet  setup.py install
```

Comment:
This cannot work as you added pip arguments to a classic `python setup.py install` script. Those are not supported.

You can convert the `python setup.py install` to `pip install .` and add the arguments.